package.json

```diff
@@ -36,8 +36,7 @@
   },
   "husky": {
```

Comment:
Remove also the `keys:decrypt` and `keys:encrypt` scripts? (and associated `.sh` files)